### PR TITLE
parameterize slack notifications

### PR DESCRIPTION
**Pull request by jacmrob**
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by jacmrob** [comment]
jenkins retest

**Comment by jacmrob** [comment]
jenkins retest

**Comment by jacmrob** [comment]
jenkins retest

**Review by breilly2 — APPROVED** [review]
:+1: Not sure exactly what everything is doing but I trust that it works. Also assuming that SLACK_CHANNEL comes from Jenkins somehow.

File: automation/docker/run-tests.sh [hunk 1]
```diff
@@ -77,7 +77,7 @@ docker run --rm -e VAULT_TOKEN=${VAULT_TOKEN} \
     broadinstitute/dsde-toolbox:dev render-templates.sh
 
 if [ "$DOCKERHOST" != "" ]; then
-    HOST_MAPPING="--add-host=firecloud-fiab.dsde-${ENV}.broadinstitute.org:${DOCKERHOST} --add-host=firecloud-orchestration-fiab.dsde-${ENV}.broadinstitute.org:${DOCKERHOST} --add-host=rawls-fiab.dsde-${ENV}.broadinstitute.org:${DOCKERHOST} --add-host=thurloe-fiab.dsde-${ENV}.broadinstitute.org:${DOCKERHOST} --add-host=sam-fiab.dsde-${ENV}.broadinstitute.org:${DOCKERHOST} -e SLACK_API_TOKEN=$SLACK_API_TOKEN -e BUILD_NUMBER=$BUILD_NUMBER -e TEST_CHANNEL=dsde-qa"
+    HOST_MAPPING="--add-host=firecloud-fiab.dsde-${ENV}.broadinstitute.org:${DOCKERHOST} --add-host=firecloud-orchestration-fiab.dsde-${ENV}.broadinstitute.org:${DOCKERHOST} --add-host=rawls-fiab.dsde-${ENV}.broadinstitute.org:${DOCKERHOST} --add-host=thurloe-fiab.dsde-${ENV}.broadinstitute.org:${DOCKERHOST} --add-host=sam-fiab.dsde-${ENV}.broadinstitute.org:${DOCKERHOST} -e SLACK_API_TOKEN=$SLACK_API_TOKEN -e BUILD_NUMBER=$BUILD_NUMBER -e SLACK_CHANNEL=${SLACK_CHANNEL}"
```
Comment (andy7i) [inline automation/docker/run-tests.sh:80]: is `${SLACK_CHANNEL}` defined somewhere?

File: automation/test.sh [hunk 2]
```diff
@@ -11,7 +11,13 @@ sbt clean
 
 cat testout.txt | sed -n -e '/Run completed/,$p' | sed $'s,\x1b\\[[0-9;]*[a-zA-Z],,g' > output/testsummary.txt
 
-curl -F file=@output/testsummary.txt -F channels=#${TEST_CHANNEL} -F token=${SLACK_API_TOKEN} -F filename=fiab-test-${BUILD_NUMBER} https://slack.com/api/files.upload
+if [[ $TEST_EXIT_CODE != 0 ]]; then COLOR=danger; else COLOR=good; fi
+BUILD_NAME=fiab-test-${BUILD_NUMBER}
+
+ATTACHMENTS='[{"color":"'"${COLOR}"'","text":"'"${BUILD_NAME}"'"]'
+
+curl -F channels=#${SLACK_CHANNEL} -F token=${SLACK_API_TOKEN} -F attachments=${ATTACHMENTS} https://slack.com/api/chat.postMessage
```
Comment (andy7i) [inline automation/test.sh]: does this work without inputs to `channels` ?

Comment (andy7i) [reply automation/test.sh]: (since it is commented out)

File: automation/docker/run-tests.sh (hunk 1 above)
Comment (jacmrob) [reply automation/docker/run-tests.sh:80]: it comes in through jenkins if we are running through jenkins.  if it's blank it won't break anything, it just won't publish to slack

File: automation/test.sh (hunk 2 above)
Comment (jacmrob) [reply automation/test.sh]: it only looks commented out; it's actually  `#` character in the option

**Comment by jacmrob** [comment]
@breilly2 go back to vacation :)

**Review by andy7i — APPROVED** [review]

**Pull request merged**